app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  has_many :posts, foreign_key: 'author_id'
+
+  # attribute
+  attribute :name, :string
+  attribute :bio, :text
+  attribute :posts_count, :integer, default: 0
+  attribute :photo, :string
```

Comment:
- [ ] **[OPTIONAL]** Is there really any need to specify these attributes here? Best case we can add them as a comment at the top of the file, as would be done by the `annotate` gem.